.gitignore

```diff
@@ -249,7 +249,6 @@ ClientBin/
 *.dbmdl
 *.dbproj.schemaview
 *.jfm
-*.pfx
```

Comment:
we actually use pfx files in this repo (certificates used by networking benchmarks)